src/component/recipe/RecipeIngredient.jsx

```diff
@@ -10,11 +10,10 @@ import SpacingDesign from '../context/design/SpacingDesign';
 import LogStatus from '../context/auth/LogStatus';
 import backgroundBG from '../../assets/lowpoly2.png';
 
-const RecipeIngredient = () => {
+const RecipeIngredient = ({ingredients}) => {
 
-  const faykeIngeredients = ['potato', 'bacon', 'cheese', 'broccoli', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla', 'farfalla'];
-  const firstHalfFaykeIng = faykeIngeredients.slice(0, Math.ceil(faykeIngeredients.length / 2));
-  const secondHalfFaykeDataIng = faykeIngeredients.slice(Math.ceil(faykeIngeredients.length / 2));
+  const firstHalfFaykeIng = ingredients.slice(0, Math.ceil(ingredients.length / 2));
```

Comment:
I'm assuming this will need to be changed when real data is brought in. Try to make notes of where these changes need to occur so they can be found easily.